common/changes/@uifabric/experiments/miwhea-export-fluent-styles_2018-06-21-16-25.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/experiments",
+      "comment": "Export Fluent constants, theme, component styles, and documentation",
+      "type": "patch"
```

Comment:
minor because new stuff were added to API surface (the root folder)

Reply:
Fixed.